apps/bot/.env.example

```
@@ -1,26 +1,40 @@
 #
 # APP
 #
-NODE_ENV=development
+NODE_ENV=production
+PORT=3001
+HOST=localhost
+
 
 #
-# AUDIO_RECORDER
+#ZOOM
 #
-CHUNK_DURATION=10
-FFMPEG_PATH=/usr/bin/ffmpeg
-OUTPUT_DIRECTORY=/home/ubuntu/audio
+
+MEETING_ID=1
+MEETING_LINK=https://us05web.zoom.us/j/84428041453?pwd=HDfwVSbpkCffotNDs7t43jblPNbdFQ.1gHtwrJ
+MEETING_PASSWORD=gHtwrJ
+BOT_NAME=Meetlytic
 
 #
 # OPEN_AI
 #
-OPEN_AI_KEY=key
-TRANSCRIPTION_MODEL=whisper-1
-TEXT_GENERATION_MODEL=gpt-4o-mini
+
+OPEN_AI_KEY=dsf
+TRANSCRIPTION_MODEL=whisper
+TEXT_GENERATION_MODEL=ffs
```

```suggestion
OPEN_AI_KEY=key
TRANSCRIPTION_MODEL=whisper-1
TEXT_GENERATION_MODEL=gpt-4o-mini
```
Avoid unnecessary changes.